src/atoms/visual-story/web-story-page-components/web-story-page-components.tsx

```diff
@@ -1,16 +1,22 @@
 import React, { Fragment } from "react";
+import atob from "atob";
 import { StoryElement, Image } from "../../../atoms";
 import { WebStoryPageComponentsTypes, AnimationTypes } from "./types";
 import { getFigcaptionText } from "../../../molecules/hero-image/hero-image";
 import styled from "styled-components";
 import { withStoryAndConfig } from "../../../context";
 import { getAnimationProps } from "./web-story-page-components.helpers";
 import get from "lodash/get";
+import { VideoWebStory } from "../../video-web-story";
 
 const WebStoryPageComponentsBase = ({ card, config, story }: WebStoryPageComponentsTypes) => {
   const titleElement = card["story-elements"].find((el) => el.type === "title");
   const textElements = card["story-elements"].filter((el) => el.type === "text" && el.subtype !== "cta");
   const imageElement = card["story-elements"].find((el) => el.type === "image");
+
+  const videoElement = card["story-elements"].find((el) => el.type === "jsembed");
```

Comment:
https://github.com/quintype/ace-planning/issues/676
```
- we'll parse the jsembed and see if it has a .mp4 or other such supported video extension (check which all videos are supported)
- if it is a supported video, we need to add it as a video in the web story template
```

We can't assume jsembed == video. There needs to be a check

Reply:
We already checked this in the canTakeCard.

Reply:
missed that. Looks ok